// Copyright 2018 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// See platform2/media_perception/mojom/common.mojom for relevant comments on
// data fields.

syntax = "proto2";

package mri;

message NormalizedBoundingBox {
  optional float x_min = 1;
  optional float y_min = 2;
  optional float x_max = 3;
  optional float y_max = 4;
  optional uint32 normalization_width = 5;
  optional uint32 normalization_height = 6;
}

enum DistanceUnits {
  UNITS_UNKNOWN = 0;
  METERS = 1;
  PIXELS = 2;
}

message Distance {
  optional DistanceUnits units = 1;
  optional float magnitude = 2;
}
